README.md

```diff
@@ -1,2 +1,45 @@
 # chpl-api-for-dummies
 Provides an example of an application that calls the CHPL REST Service
+
+# Installation instructions
+
+## Install required software
+
+Java 1.8.0
+Eclipse (latest)
+
+## Clone the repository
+Using Git Bash, run the following command:
+
+```sh
+$ git clone https://github.com/chpladmin/chpl-api.git
+```
+
+## Add external JARs to project
```

Comment:
Is there a way to set this up as a Maven project or something? Manually adding jars is not a super convenient way of building a project